app/sidekiq/benefits_intake_status_job.rb

```diff
@@ -25,24 +25,24 @@ def perform
     resolved_form_submission_ids = form_submissions_and_attempts
                                    .where(form_submission_attempts: { aasm_state: %w[vbms failure] })
                                    .pluck(:id)
-    pending_form_submissions = form_submissions_and_attempts
-                               .where(form_submission_attempts: { aasm_state: 'pending' })
-                               .where.not(id: resolved_form_submission_ids)
-    # We're calculating the resolved_form_submissions and removing them because it is possible for a FormSubmission
+    pending_form_submission_attempts = FormSubmissionAttempt
+                                       .where(aasm_state: 'pending')
+                                       .where.not(form_submission_id: resolved_form_submission_ids)
```

Comment:
I do not think you would need line 30, which would then also allow you to eliminate 24-27; not querying for all of the ids of ‘finished’ submissions.  There could be pending attempts for a submission that previously failed, especially with the recent change to FormSubmission and FormSubmissionAttempt

Reply:
Good point! And in reality, we'll probably never have a submission attempt in pending and _then_ pile another attempt on top of it which could go to vbms or failure. Even if we did we'd want to keep an eye on the pending one until it resolves. Thanks!